shell/monitor/monitor_adla.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+declare -A _ADLA_test_ids
+
+_monitor_ADLA(){
+
+  local experiment_id=$1 adla_account=$2 current_job_id job_id output
+
+  # currently running test (ADLA UUID)
+  current_job_id=$(_monitor_get_test_id)
+ 
+  # remember all jobs seen so far, to always emit the status for everything seen so far
+  _ADLA_test_ids[${current_job_id}]=1
+  
+  for job_id in "${!_ADLA_test_ids[@]}"; do
+
+    output=$(az dla job show --output json --account "${adla_account}" --job-identity "${job_id}")
+
+    if [[ ! "$output" =~ (^(null|\{\})$|^$) ]]; then
```

Comment:
I know that this was a response to one of my earlier comments but if we don't need it, then it can be removed.